pyfakefs/fake_filesystem.py

```diff
@@ -2915,6 +2924,21 @@ def _add_standard_streams(self) -> None:
         self._add_open_file(StandardStreamWrapper(sys.stdout))
         self._add_open_file(StandardStreamWrapper(sys.stderr))
 
+    def _create_temp_dir(self):
+        if not self.create_temp_dir:
+            return
+        # the temp directory is assumed to exist at least in `tempfile`,
+        # so we create it here for convenience
+        temp_dir = tempfile.gettempdir()
+        if not self.exists(temp_dir):
+            self.create_dir(temp_dir)
+        if sys.platform != "win32" and not self.exists("/tmp"):
+            # under Posix, we also create a link in /tmp if the path does not exist
+            self.create_symlink("/tmp", temp_dir)
```

Comment:
I still don't understand why this added symlink is required.

Reply:
Well, the issue author mentioned the [FHS](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch03s18.html), which was good enough for me, though I may be wrong.